### PR TITLE
Enhancements/de duplication

### PR DESCRIPTION
**Pull request by kruai**
De-duplicate on POST and PUT requests.

### CODE REVIEW

File: api/tags.py [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
 from insights_connexion.db.gino import db
+from sqlalchemy import and_
 
 
 async def _get_one_tag(id):
     return await Tag.query.where(Tag.id == id).gino.first()
 
+async def _get_duplicate_tag(account_id, namespace, name, value):
+    tags = await Tag.query.where(and_(Tag.account_id == account_id, Tag.namespace == namespace, Tag.name == name, Tag.value == value)).gino.all()
+    return await Tag.query.where(and_(Tag.account_id == account_id, Tag.namespace == namespace, Tag.name == name, Tag.value == value)).gino.first()
```
Comment (tahmidefaz) [inline api/tags.py]: @kruai what is the `tags` variable doing here?

**Review by tahmidefaz — COMMENTED** [review]
Check my comment below! :smile:

File: api/tags.py (hunk 1 above)
Comment (kruai) [reply api/tags.py]: Oops, that's a leftover from debugging. I'll remove it!

**Review by tahmidefaz — APPROVED** [review]
Looks good! :+1:

**Pull request merged**